### DISCUSSION TITLE
My IP appears to be blocked from all vercel-hosted sites and from vercel itself

### DISCUSSION THREAD

**Discussion by melefabrizio** [vercel/community#7314] [Account Issue] [answered]
### Question

Hello, it seems that since yesterday all sites hosted on vercel, and vercel itself, are not reachable from my home ip address. If I switch to a mobile hotspot, to the neighbours wifi, or if I turn on a VPN everything is reachable again.

I tried to run the `vercel-support/vercel-connect-debug` script and all ip addresses save for `76.76.21.22` have ping timeout, and their traceroute stops a few steps in:
```
traceroute to 76.76.21.21 (76.76.21.21), 30 hops max, 48 byte packets
 1  192.168.1.1 (192.168.1.1)  7.797 ms  2.506 ms  2.372 ms
 2  62.97.33.70 (62.97.33.70)  6.220 ms  5.945 ms  5.095 ms
 3  host-x-x-x-x.pool.intred.it (x-x-x-x)  6.146 ms  6.687 ms  5.539 ms
 4  * * *
 5  62.97.33.135 (62.97.33.135)  10.483 ms  6.034 ms  5.455 ms
 6  62.97.33.75 (62.97.33.75)  7.111 ms  12.072 ms  37.988 ms
 7  amazon.mix-it.net (217.29.66.74)  7.383 ms  7.834 ms  38.437 ms
 8  * * *
 [ ... ]
 30 * * *
```

Is not a DNS problem, as all domains of all websites affected (nextjs.org, vercel.com, my own app spranga.xyz, python.langchain.com, and more) are correctly resolved by my resolver using Cloudflare or Google DNS.

Can anyone help me?

**Comment by pawlean** [comment]
Hi @melefabrizio,

Could you please open a case [here](https://vercel.com/help)? Scroll down, click on "Get Help," and follow the steps provided. This issue is account-specific, so we can't address it in a public forum.

Thank you!

**Comment by melefabrizio** [comment]
I cannot open a case on this issue, as I do not have a paid account.

**Reply by pawlean** [reply]
Apologises! Let me look into this for you 🙏🏼

**Reply by pawlean** [reply]
Hi @melefabrizio! 

I've sent you an email on this to dig deeper on the `vercel-support/vercel-connect-debug` script output.

This is typically not a block from us, but an ISP or network policy issue. You could consider contacting your ISP directly, as they may respond more quickly.

**Reply by melefabrizio** [reply]
Hi, any news on the support ticket we opened via email? It's been two weeks, and still half of the dev-related internet times out.

**Reply by pawlean** [reply]
Hi @melefabrizio!

Apologises, I passed this onto my colleague but didn't follow up. Let me reach out again.

I also wanted to see, did you check if this was a possible ISP / network policy issue?

> This is typically not a block from us, but an ISP or network policy issue. You could consider contacting your ISP directly, as they may respond more quickly.

**Reply by melefabrizio** [reply]
Hello, I'm waiting for a reply from my ISP, but my neighbour, which uses the same ISP but has a different public ip address, can access vercel with no problems. My local network has no policies in place.

Please note that I cannot use any vercel hosted website or service from my home, and my work requires me to use both vercel and vercel hosted websites

**Reply by pawlean** [reply]
@melefabrizio We've identified an issue and will get back to you as soon as we can. 🙏🏼 I really appreciate your patience!

**Reply by pawlean** [reply]
Hi, @melefabrizio! Still looking into this I'm afraid, thank you for your patience.

**Comment by muridil** [comment]
Hello, the same issue is happening to me, my public IP is 109.81.116.218 (ISP: O2 Czech Republic A.S.) and I can't connect to either 76.76.21.61 or 76.76.21.22
Can you pls look into it?

**Reply by pawlean** [reply]
Hi @muridil!

Can you follow the steps in our new community? https://vercel.community/t/resolving-ip-blocking-issues/171

**Comment by pawlean** [comment, marked as answer]
Closing this. Please refer to our new community platform to help resolve this: https://vercel.community/t/resolving-ip-blocking-issues/171

[a comment by an automated account is not shown]
